param/__init__.py

```diff
@@ -1014,6 +1014,10 @@ def _validate(self, val):
                              (self.name,len(val),self.length))
 
 
+    @classmethod
+    def ir_deserialize(cls, value):
```

Comment:
My preference would be to keep these as `serialize` and `deserialize` to keep the API simple.

Reply:
I have no real preference here and I am happy to revert this change. @jbednar ?

Reply:
No preference.

Reply:
Ok, in that case Philipp wins the vote and I'll revert!

Reply:
Done in 191d431